code/datums/controllers/action_controls.dm

```diff
@@ -1544,16 +1544,23 @@ var/datum/action_controller/actions
 			interrupt(INTERRUPT_ALWAYS)
 			return
 
+	onInterrupt()
+		..()
+		if (target?.butcherer == owner)
```

Comment:
is this a thing that can happen???
how can a dead thing do anything, let alone butcher itself

Reply:
`target.butcherer` is the last thing that started butchering the target, here we check if it's equal to owner to stop the immediate interrupt on trying to butcher something that's already being butchered from clearing the value of the `butcherer` var
hope that's coherent enough